extensions/spotify-player/CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# How to Contribute
+
+🎉 First of all, thanks for taking the time to contribute! 🎉
+
+Please follow the [Contribute to an Extension](https://developers.raycast.com/basics/contribute-to-an-extension) guide.
+
+We're still figuring things out and updating our guidelines. If at any point something is unclear, please tell us in [our community](https://raycast.com/community).
+
+---
+
+### Wallpapers
+
+Screenshots are taken with the "[Moonrise](https://www.raycast.com/uploads/wallpapers/moonrise.png)" background from https://www.raycast.com/wallpapers
```

Comment:
I couldnt find the one previously used but thought it would make sense to use something from the public set? Just to make future screenshots safer/easier

Reply:
yeah, don't worry about it 👍 
hm, interesting why the one I used is not a part of the wallpapers set 🤔